pkg/config/database/config.go

```diff
@@ -0,0 +1,52 @@
+package database
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+type DBUse string
+
+func (u DBUse) ToLower() DBUse {
+	return DBUse(strings.ToLower(string(u)))
+}
+
+const (
+	MongoDB DBUse = "mongoDB"
+	CqlDB   DBUse = "cqlDB"
+)
+
+type DBConfig interface {
+	Validate() error
+}
+
+type Config[MongoConfig DBConfig, CQLDBConfig DBConfig] struct {
+	Use     DBUse       `yaml:"use" json:"use"`
```

Comment:
Do we need this struct at all? Why not to skip it completely, and work only with - if url is set on mongo db, if not, if url is set on scyllaDB, and decide based on this. What do you think?

Reply:
Because both part could be configured and with this option you can simple change DB. Yes it is more helpful struct for future.

Reply:
I don't see a use case to configure both databases. There is no real world scenario I can come up with. You?

Reply:
You need to choose which you want to use it. It is much better when you dont have have property with two meanings (configured = host )like URL/HOST. 
Eg migration - if we support it in future then you need to have congifured both values + direction.